warp10/src/main/sh/warp10.sh

```diff
@@ -212,6 +212,18 @@ postInit() {
 " >>"${WARP10_CONFIG_DIR}/99-init.conf"
   ${JAVACMD} -cp "${WARP10_JAR}" -Dfile.encoding=UTF-8 io.warp10.GenerateCryptoKeys ${TEMPLATE} >> "${WARP10_CONFIG_DIR}/99-init.conf"
 
+
+  echo "
+//
+// Uncomment the following lines to enable metrics collections.
+// Report here the result of the token generation with the envelope ${WARP10_HOME}/tokens/sensision-tokengen.mc2
+//
+// SENSISIONEVENT, SENSISIONGET, SENSISIONSET, SENSISIONUPDATE
+#warpscript.extension.sensision = io.warp10.script.ext.sensision.SensisionWarpScriptExtension
+#sensisionReadToken@/sensision=
```

Comment:
be more specific in the path, only allowing the script we added to access the tokens